site2/docs/functions-cli.md

```diff
@@ -1,197 +1,83 @@
+| secrets | N/A| Map<String,Object> | `--secrets`	| The mapping from secretName to objects that encapsulate how the secret is fetched by the underlying secrets provider. |
+| runtime | N/A| String | N/A | The runtime of a function. Available values: `java`,`python`, `go`. |
+| autoAck | N/A| Boolean | `--auto-ack` | Whether the framework acknowledges messages automatically or not. |
+| maxMessageRetries | N/A| Int |	`--max-message-retries` | The number of retries to process a message before giving up. |
+| deadLetterTopic | N/A| String | `--dead-letter-topic` | The topic used for storing messages that are not processed successfully. |
+| subName | N/A| String | `--subs-name` | The name of Pulsar source subscription used for input-topic consumers if required.|
+| parallelism | N/A| Int | `--parallelism` | The parallelism factor of a function, that is, the number of function instances to run. |
+| resources | N/A | Resources	| N/A| N/A	 |
+| | cpu | double | `--cpu` | The CPU in cores that need to be allocated per function instance (for Kubernetes runtime only). |
+| | ram | Long | `--ram` | The RAM in bytes that need to be allocated per function instance (for process/Kubernetes runtime only). |
+| | disk | Long | `--disk` | The disk in bytes that need to be allocated per function instance (for Kubernetes runtime only). |
+| fqfn | N/A | String | `--fqfn` | The Fully Qualified Function Name (FQFN) of a function. |
+| windowConfig | | | WindowConfig | N/A| N/A|		
+| | windowLengthCount | Int | `--window-length-count` | The number of messages per window. |
+| | windowLengthDurationMs | Long | `--window-length-duration-ms` | The time duration (in milliseconds) per window. |
+| | slidingIntervalCount | Int | `--sliding-interval-count` | The number of messages after which a window slides. |
+| | slidingIntervalDurationMs | Long | `--sliding-interval-duration-ms` | The time duration after which a window slides. |
+| | lateDataTopic | String | N/A	| N/A |
+| | maxLagMs | Long | N/A	| N/A |
+| | watermarkEmitIntervalMs | Long | N/A	| N/A |
+| | timestampExtractorClassName | String | N/A	 | N/A|
+| | actualWindowFunctionClassName | String | N/A	| N/A |
+| timeoutMs | N/A| Long | `--timeout-ms` | The message timeout in milliseconds. |
+| jar | N/A| String | `--jar` | The path of the JAR file for a function (if the function is written in Java). It also supports URL paths that workers can download the package from, including HTTP, HTTPS, file (file protocol assuming that file already exists on worker host), and function (package URL from packages management service). |
+| py | N/A| String | `--py` | The path of the main Python/Python wheel file for a function (if the function is written in Python). It also supports URL paths that workers can download the package from, including HTTP, HTTPS, file (file protocol assuming that file already exists on worker host), and function (package URL from packages management service).  |
```

Comment:
is ` (if the function is written in Python)` redundant because it already says `... main Python/Python wheel file ...`?